auth/realm/base/src/main/java/org/wildfly/security/auth/realm/ElytronMessages.java

```diff
@@ -153,6 +153,21 @@ interface ElytronMessages extends BasicLogger {
     @Message(id = 13006, value = "Filesystem-backed realm unable to encrypt identity")
     RealmUnavailableException fileSystemRealmEncryptionFailed(@Cause Throwable cause);
 
-    @Message(id = 13007, value = "Filesystem-backed realm found an incompatible identity version. Requires at least version: %s")
-    RealmUnavailableException fileSystemRealmIncompatibleIdentityVersion(String expectedVersion);
+    @Message(id = 13007, value = "Signature for the following identity is invalid: %s.")
```

Comment:
We shouldn't remove the existing message. Looks like the id here should be 13008 and the ones below need to be incremented too.

Reply:
Just checked and looks like this message was left over in the encryption rfe even though it's not being used anywhere. I think it's safe to delete this.